flowmachine/tests/test_cache.py

```diff
@@ -88,11 +85,7 @@ def test_store_cache_multi(flowmachine_connect):
     hl1.store().result()
     # Should be stored
     assert hl1.is_stored
-    in_cache = bool(
-        flowmachine_connect.fetch(
-            f"SELECT * FROM cache.cached WHERE query_id='{hl1.md5}'"
-        )
-    )
+    in_cache = cache_table_exists(flowmachine_connect, hl1.md5)
 
     assert in_cache
```

Comment:
Should this also check that the underlying daily_location now exists in the cache? Apologies if this is actually checked in a different test further down which I haven't seen yet.